src/python/module/smartredis/util.py

```diff
@@ -64,7 +71,7 @@ def metadata_from_numpy(array):
         raise TypeError(f"Incompatible metadata type provided {dtype}")
 
     @staticmethod
-    def from_string(type_name):
+    def from_string(type_name: str)->type:
```

Comment:
There is very, very, very subtle difference (that I do not fully remember) between `type` and `t.Type` when used without `[]`. I'm also not fully confident that that this syntax is accepted by py3.7 or 3.8 (3.9 was when the `list[T] == t.List[T]` syntax was introduced, I imagine that's true for `type` as well).

For those reasons, just for safety, can we change the return type here to `t.Type`

Reply:
done